### ISSUE TITLE
Code review

### ISSUE THREAD

**Issue by DmitrijsSemjonovsDP13** [DavisMartuzans/ToDoList#1]
Īsti darbs tikai sākts , jo ir tikai priekš izskats  ( poga , fons , tekst ) , bet poga tomēr nestrādā (Var koda  redzēt , ka cilvēks jau sāka veidot to pogu !). Darbs vnk , nav pabeigts un viss , damaju , ka šim cilvēkam sanāks diezgan labs un smuks saits . Bet lūdzu izlabo pašu kodu , kā minimums nomainī vārdus  , nevajag copy past no skolotāja .
乁 ˘ o ˘ ㄏ

**Comment by DavisMartuzans** [comment]
Yes izdarīšu, vienkārši pilnīgi aizmirsu par darbu bet visu pārmainīšu.
